hooks/msghooks.py

```diff
@@ -120,6 +120,11 @@ def receive_message():
                 return f"{reply} Here you go: 🍪"
             else:
                 return f"{reply} No."
+        if re.match(r"sudo (pl(s|z|ease) )?make meh? a coo?kie", without_ping):
+            if not (message["user_id"] == 281362 or message["user_id"] == 497390 or message["user_id"] == 475866):
```

Comment:
oh dear you seem to have accidentally put a `not` here...

Reply:
nah its on purpose

Reply:
no I'm pretty sure you're wrong ;)

I'm in charge here!

Reply:
bruh